x-pack/plugins/cases/server/client/attachments/get.ts

```diff
@@ -115,32 +117,40 @@ export const getAllAlertsAttachToCase = async (
   clientArgs: CasesClientArgs,
   casesClient: CasesClient
 ): Promise<AlertResponse> => {
-  const { unsecuredSavedObjectsClient, authorization, attachmentService } = clientArgs;
-
-  // This will perform an authorization check to ensure the user has access to the parent case
-  const theCase = await casesClient.cases.get({
-    id: caseId,
-    includeComments: false,
-    includeSubCaseComments: false,
-  });
-
-  const { filter: authorizationFilter, ensureSavedObjectsAreAuthorized } =
-    await authorization.getAuthorizationFilter(Operations.getAlertsAttachedToCase);
-
-  const alerts = await attachmentService.getAllAlertsAttachToCase({
-    unsecuredSavedObjectsClient,
-    caseId: theCase.id,
-    filter: authorizationFilter,
-  });
-
-  ensureSavedObjectsAreAuthorized(
-    alerts.map((alert) => ({
-      owner: alert.attributes.owner,
-      id: alert.id,
-    }))
-  );
-
-  return normalizeAlertResponse(alerts);
+  const { unsecuredSavedObjectsClient, authorization, attachmentService, logger } = clientArgs;
+
+  try {
```

Comment:
I noticed this wasn't wrapped in a `try/catch` so just adding that and indenting.